packages/bun-types/bun.d.ts

```diff
@@ -962,6 +967,44 @@ declare module "bun" {
       | "entry-point";
   }
 
+  type ModuleFormat = "esm"; // later: "cjs", "iife"
+
+  interface BuildConfig {
+    entrypoints: string[]; // list of file path
+    target?: Target; // default: "browser"
+    // module?: ModuleFormat; // later: "cjs", "iife"
+    outdir?: string; // output directory
+
+    naming?: {
+      chunk?: string;
+      entrypoint?: string;
+    }; // | string;
+    // root?: string; // project root
+    // transform?: boolean; // default: false, transform instead of bundling
+    splitting?: boolean; // default true, enable code splitting
+    plugins?: BunPlugin[];
+    // manifest?: boolean; // whether to return manifest
+    external?: Array<string | RegExp>;
+    // origin?: string; // e.g. http://mydomain.com
+    // assetOrigin?: string; // e.g. http://assets.mydomain.com
+    // loaders?: { [k in string]: Loader };
+    sourcemap?: "none" | "inline" | "external"; // default: "none"
+    minify?:
+      | boolean
+      | {
+          whitespace?: boolean;
+          syntax?: boolean;
+          identifiers?: boolean;
+        };
+    treeShaking?: boolean;
+  }
+
+  type BuildResult<T> = {
```

Comment:
can you make T into `T extends Blob = Blob` or whatever the shared type is, so that `BuildResult` on its own is a usable type.

(i was gonna say that not doing this broke one way of using the result, but that was just me not realizing that writefilesync doesn't support blobs)